LifeCycleAssessment_Engine/Query/ProjectLifeCycleAssessmentResults.cs

```diff
@@ -0,0 +1,166 @@
+
+            string lateralSys = System.Convert.ToString(lcaResult.LifeCycleAssessmentScope.LateralStructuralMaterial);
+            if (lateralSys == "Undefined")
+                BH.Engine.Reflection.Compute.RecordNote("Please enter your primary lateral structural system within the LifeCycleAssessmentScope.");
+
+            string lcaPhases = "";
+            if(lcaResult.LifeCycleAssessmentScope.LifeCycleAssessmentPhases.Count() >= 1)
+            {
+                foreach (var c in lcaResult.LifeCycleAssessmentScope.LifeCycleAssessmentPhases)
+                    lcaPhases += c.ToString() + ",";
+                lcaPhases = lcaPhases.Remove(lcaPhases.LastIndexOf(','));
+            }
+            else
+            {
+                BH.Engine.Reflection.Compute.RecordWarning("Please enter the specific LCA phases you are reporting within the LifeCycleAssessmentScope.");
+                lcaPhases = "Undefined";
+            }
+
+            string lod = System.Convert.ToString(lcaResult.LifeCycleAssessmentScope.LevelOfDevelopment);
+            if (lod == "Undefined")
+                BH.Engine.Reflection.Compute.RecordNote("Please enter your project's level of development upon reporting within the LifeCycleAssessmentScope.");
+
+            string projectId = lcaResult.LifeCycleAssessmentScope.AdditionalNotes;
+            if(lcaResult.LifeCycleAssessmentScope.AdditionalNotes == "")
+            {
+                BH.Engine.Reflection.Compute.RecordNote("Please enter your project's identifier the LifeCycleAssessmentScope AdditionalNotes.");
+                projectId = "Undefined";
+            }           
+
+            object location = lcaResult.LifeCycleAssessmentScope.Location;
+            string projectLocation = "";
+            if (location != null)
+            {
+                string latitude = System.Convert.ToString(lcaResult.LifeCycleAssessmentScope.Location.Latitude);
+                string longitude = System.Convert.ToString(lcaResult.LifeCycleAssessmentScope.Location.Longitude);
+                projectLocation = String.Concat(latitude, longitude + ",");
+            }
+            else
+            {
+                projectLocation = "Undefined";
+                BH.Engine.Reflection.Compute.RecordWarning("Please enter your project's location in Lat Long format within the LifeCycleAssessmentScope.");
+            }
+
+            string projectName = lcaResult.LifeCycleAssessmentScope.ProjectName;
+            if (projectName == "")
+            {
+                BH.Engine.Reflection.Compute.RecordWarning("Please enter your project name within the LifeCycleAssessmentScope.");
+                projectName = "Undefined";
+            }
+
+            string projectType = System.Convert.ToString(lcaResult.LifeCycleAssessmentScope.ProjectType);
+            if (projectType == "Undefined")
```

Comment:
```suggestion
            if (projectType.ToLower() == "undefined")
```

As above, suggestion.

Reply:
See above reasoning, will adjust if needed.